src/network/network_gui.cpp

```diff
@@ -2073,13 +2074,14 @@ struct NetworkClientListWindow : Window {
 				break;
 
 			case WID_CL_SERVER_VISIBILITY:
-				SetDParam(0, _server_visibility_dropdown[_settings_client.network.server_advertise]);
+				SetDParam(0, STR_NETWORK_SERVER_VISIBILITY_LOCAL + _settings_client.network.server_game_type);
 				break;
 
-			case WID_CL_SERVER_JOIN_KEY:
+			case WID_CL_SERVER_JOIN_KEY: {
```

Comment:
Does adding this here mean that all previously commits in between fail to build? That would not be very nice for a bisect session.

Reply:
Pfft, only for MSVC users. Who cares about those anyway? :P (sometimes it is hard to put things in the right commit when you have this many .. let me try again :D)